Aztec/Classes/Libxml2/DOM/Data/ElementNode.swift

```diff
@@ -199,7 +199,15 @@ public class ElementNode: Node {
     /// - Returns: `true` if this is a block-level element.  `false` otherwise.
     ///
     public func isBlockLevel() -> Bool {
```

Comment:
Why not make this function and the next two readyonly calculated vars?

Reply:
I only use computed properties with algorythms with `O(1)` complexity.  That way the caller knows that whenever they're accessing a computed property there won't be extra processing and they can access it multiple times without big trouble.

This method contains a call to `Swift.Array.contains()` which has a `O(n)` complexity, so I'd prefer the caller to know that it's best to avoid calling this multiple times in a row.